Portfolio/index.css

```diff
@@ -0,0 +1,772 @@
+/* navigation Bar */
+
+*{
+    margin: 0px;
+    padding: 0px;
+    border: 0px;
+
+}
+body{
+    background-color:#004572;
+}
+.navbar {
+    position: relative;
+    
+    cursor: url('/cursor.png'),auto;
+ /* position: relative; */
+ 
+ /* border: 2px solid red; */
+ width: 96.3%;
+    display: inline-flex;
+    align-items: center;
+    justify-content: space-between; 
+    padding: 10px 23px;
+    background-color:#003E66;
+    color: #00D8FF;
+    position: relative;
+}
+
+.navbar h1 {
+    font-family: Revalia;
+    margin: 0;
+    font-size: 2rem;
+}
+
+.navbar ul {
+    list-style-type: none;
+    margin: 0;
+    padding: 0;
+    display: flex;
+    padding: 1px 215px;
+}
+
+.navbar ul li {
+    margin-right: 20px;
+}
+
+.navbar ul li:last-child {
+    margin-right: 0;
+}
+
+.navbar ul li a {
+    font-family: Revalia;
+    color: #00D8FF;
+    text-decoration: none;
+    padding: 0 23px;
+}
+
+.navbar ul li a:hover {
+    text-decoration: underline;
+}
+
+.phnphn img{
+    /* padding: 0px 0px; */
+    margin-left: 1px;
+   margin-top: 10px;
+}
+.phnbtn button  {
+    border-radius: 90px;
+    padding: 6px 12px;
+    background:transparent ;
+    border: 1px solid  #00D8FF;
+}
+.phnbtn  strong{
+    background:transparent;
+    color: #00D8FF;
+    font-family: Roboto;
+ width: 34px;
+    padding: 8px 16px;
+ 
+    /* border: 1px solid #00D8FF; */
+  
+    cursor: pointer;
+}
+
+
+.navbar img {
+    margin-left: 10px;
+ 
+}
+
+
+
+
+.abtme{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    background-color: #004572;
+  
+    position: relative;
+    height: 500px;
+}
+.abtme img{
+    
+    margin-top: 66px;
+   
+}
+.abtme h1{
+    margin-left: 100px;
+    display: flex;
+    color:  #00D8FF;
+    
+    font-size: 3rem;

```

Comment:
Consider organizing the properties in a more structured way for better readability.